tests/data-access/test_wdqs_reader.py

```diff
@@ -5,9 +5,11 @@
 
 
 class MockResponse:
-    @staticmethod
-    def convert():
-        return {"results": {"bindings": [{'a': 'b'}]}}
+    def __init__(self, res=[{'a': {'value': 'b'}}]):
+        self.res = res
+
+    def convert(self):
+        return {"results": {"bindings": self.res}}
```

Comment:
Looks like a good change to customise the MockResponse

Reply:
I was thinking of centeralizing the mocks, at least some of them. What do you think?